tests/unit/tuf_repository_service_worker/models/targets/test_crud.py

```diff
@@ -211,13 +211,24 @@ def test_update(self, monkeypatch):
         assert fake_datetime.now.calls == [pretend.call()]
 
     def test_update_to_published(self, monkeypatch):
-
+        test_targets = ["path/file1", "path.file2"]
+        crud.models.RSTUFTargets = pretend.stub(
+            path=pretend.stub(
+                in_=pretend.call_recorder(lambda *a: test_targets)
+            ),
+            published="published",
+            last_update="last_update",
```

Comment:
Shouldn't you use the `fake_time` from below as you are updating it to `datetime.now()` which you mock below?

Reply:
See my comment above.